src/reaktoro_pse/tutorials/softening_acid_ro_example.ipynb

```diff
@@ -0,0 +1,1322 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Reaktoro for modeling softening and acid addition pre-treatment for desalination systems\n",
+    "\n",
+    "\n",
+    "#### Dependencies\n",
+    "* Python - Programming language\n",
+    "* Pyomo - Python package for equation-oriented modeling\n",
+    "* IDAES - Python package extending Pyomo for flowsheet modeling\n",
+    "* cyipopt - Solver necessary for use with gray box models\n",
+    "* Reaktoro-pse - Python package for building Reaktoro gray box models\n",
+    "* watertap-solvers - for cyipopt solver wrapper"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This example demonstrates how Reaktoro blocks can model pre-treatment and conditions inside different parts of a treatment process:\n",
+    "\n",
+    "1) Precipitation processes - here modeled as Lime and Soda ash softening for Calcite precipitation, Reaktoro can be configured to use any number of chemicals and user can select to form any number of solids from those available in selected databases\n",
```

Comment:
Precipitation processes - here modeled as Lime and Soda ash softening for Calcite precipitation.  Reaktoro can be configured to use any number of chemicals and users can choose the set of solids to be formed  from those available in selected databases.

Reply:
done